doc/libpmemobj/pmemobj_ctl_get.3.md

```diff
@@ -320,6 +320,18 @@ The required class identifier will be stored in the `class_id` field of the
 This function returns 0 if the allocation class has been successfully created,
 -1 otherwise.
 
+stats.enabled | rw | - | int | int | - | boolean
+
+Enables or disables runtime collection of statistics. The statistics are never
+altered after enabling.
```

Comment:
What do you mean by "are never altered after enabling"?

Reply:
The statistics are not recalculated whenever you  enable them. In other words, if you disable statistics, the operations that are performed won't count towards statistic values after you enable them again.